docs/release-notes.rst

```diff
@@ -0,0 +1,205 @@
+IPDK Kubernetes Infrastructure Offload Release Notes
+#############################################################
+
+IPDK 24.01
+************
+
+What's new in this Release
+===========================
+
+- Service Load Balancing: Addition of service type load-balancer feature.
+   Services (UDP/TCP) can be created and load-balanced to endpoints using
+   hash in the hardware. TCP and UDP flows are pinned to endpoint after
+   selection based on SYN packet.
```

Comment:
Services can be created and dynamically distributed to endpoints. For TCP, only the SYN packet goes through the load-balancing logic. Entry is added to hardware  CT table for treatment of subsequent packets. 
Support for dynamic scale-up of endpoints.